components/ProfileLink/ProfileLink.tsx

```diff
@@ -8,8 +8,13 @@ import {
   useAuth
 } from "../auth"
 import { NavLink } from "../Navlink"
-import styles from "./ProfileLink.module.css"
-
+// import styles from "./ProfileLink.module.css"
+import styled from "styled-components";
+const ProfileLinkImageWrapper = styled(Image)`
+  @media only screen and (min-width: 992px) {
```

Comment:
We discussed this in the chat, but I think we can remove the `only screen` part of the media query here  - we use it very inconsistently throughout the project and IMO we should standardize on not using it while we're touching this.

If we get rid of the screen query, we might also be able to just use Bootstrap utility classes here and skip styled components altogether. Does this look right if we just give the `Image` the className `my-lg-0 mx-lg-2`?